std/uni.d

```diff
@@ -2061,32 +2047,43 @@ public:
         sanitize(); //enforce invariant: sort intervals etc.
     }
 
+    ///
+    unittest
+    {
+        import std.algorithm : equal;
+
+        auto set = CodepointSet('a', 'z'+1, 'а', 'я'+1);
+        foreach(v; 'a'..'z'+1)
+            assert(set[v]);
+        // Cyrillic lowercase interval
+        foreach(v; 'а'..'я'+1)
+            assert(set[v]);
+        //specific order is not required, intervals may interesect
+        auto set2 = CodepointSet('а', 'я'+1, 'a', 'd', 'b', 'z'+1);
+        //the same end result
+        assert(set2.byInterval.equal(set.byInterval));
```

Comment:
This may be the offending line (the call to `equal`). It may have something to do with the Phobos makefile splitting up the unittest runs into separate groups of modules. Perhaps one or more object files are missing from the corresponding unittest runner's build command, so it failed to link in the symbols generated from an imported module.